tests/Models/AppConfig/AccountListResult.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Can we figure out how to use models from the main project?  Seems odd to have to models for tests only.

Reply:
I don't see any other classes in Azure.Mcp.Tests.Models namespace

<img width="690" height="502" alt="Image" src="https://github.com/user-attachments/assets/9da45d8d-df99-4f6d-a326-5364b0f3036f" />